### PR TITLE
teams/docs: a vote for Icemap as the reviewer

### PR DESCRIPTION
**Pull request by Oreoxmt**
As stated in the title.

### CODE REVIEW

File: votes/0714-Icemap-as-docs-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# A Vote for Icemap as Docs Reviewer
+
+## Proposal
+
+[@Icemap](https://github.com/Icemap) has actively contributed to TiDB documentation for several months. His contribution covers the documentation of [TiDB Developer Guide](https://docs.pingcap.com/tidb/stable/dev-guide-overview) in both Chinese and English. He has provided a lot of good suggestions in his review comments. I think @Icemap shows his qualification as a reviewer of the Docs Team. The details of his contributions are listed as follows:
```
Comment (qiancai) [inline votes/0714-Icemap-as-docs-reviewer.md:5]:
```suggestion
[@Icemap](https://github.com/Icemap) has actively contributed to TiDB documentation for several months. His contribution covers the documentation of [TiDB Developer Guide](https://docs.pingcap.com/tidb/stable/dev-guide-overview) in both Chinese and English. He has provided a lot of good suggestions in his review comments. I think @Icemap shows his qualification as a reviewer of the TiDB Docs. The details of his contributions are listed as follows:
```

**Review by qiancai — APPROVED** [review]
LGTM
Congratulations!

**Review by TomShawn — APPROVED** [review]
LGTM 👏🏻

File: votes/0714-Icemap-as-docs-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Deadline
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
```
Comment (Oreoxmt) [inline votes/0714-Icemap-as-docs-reviewer.md:27]:
```suggestion
## Result

Approved by 2 binding votes:

* qiancai (binding)
* TomShawn (binding)
```

**Pull request merged**